.eslintrc

```diff
@@ -2,5 +2,8 @@
   "extends": "babel",
   "env": {
     "node": true
+  },
+  "rules": {
+    "indent": 0
```

Comment:
If I indent things correctly, this PR will have an unreadable diff because the indent of most lines will change. Don't know a good solution for that.